CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ New features and improvements:
 - Add where filter to ``read_file`` (#2552)
 
 Deprecations and compatibility notes:
+- Deprecated internal GeoDataFrame attribute `_crs` has been removed (#2578).
```

Comment:
```suggestion
- Accessing the `crs` of a `GeoDataFrame` without active geometry column was deprecated and this now raises an AttributeError (#2578).
```

To keep it about the relevant public behaviour change (the private ``_crs`` is only an implementation detail for users)

Reply:
yep, makes sense.